core/iwasm/libraries/wasi-nn/README.md

```diff
@@ -71,6 +72,18 @@ docker run \
     /assets/test_tensorflow.wasm
 ```
 
+* vx-delegate for NPU (x86 simulater)
+
+```
+docker run \
+    -v $PWD/core/iwasm/libraries/wasi-nn/test:/assets wasi-nn-vx-delegate \
+    --dir=/assets \
+    --env="TARGET=gpu" \
+    /assets/test_tensorflow.wasm
```

Comment:
is the test running in vx-delegate with `--env="TARGET=gpu"`?

Reply:
So far, yes. This is the question where the external delegation should be categorised